.github/workflows/generate.yml

```diff
@@ -36,19 +46,35 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
-
+                
       - name: Run config generator script
         env:
           NETWORK: ${{ github.event.inputs.network }}
           NAMESPACE: ${{ github.event.inputs.namespace }}
           ACTIONS_ALLOW_UNSECURE_COMMANDS: 'true'
+        if: ${{ github.event.inputs.timestampMethod == 'timestamp' }}
         run: |
           echo "Generating config with genesis timestamp ${{ github.event.inputs.genesisTime }} for network ${{ github.event.inputs.network }}-${{ github.event.inputs.namespace }}"
           chmod +x ./updateGenesisTime.sh ./genesis-state-gen-dev
           ./updateGenesisTime.sh ${{ github.event.inputs.network }} ${{ github.event.inputs.namespace }} ${{ github.event.inputs.genesisTime }}
           export SHA256_SUM=$(sha256sum ./$NETWORK/$NAMESPACE/network-config.yaml)
           echo "::set-env name=SHA256_SUM::$SHA256_SUM"
-
+          
+      - name: Run config generator script with delay
+        env:
+          NETWORK: ${{ github.event.inputs.network }}
+          NAMESPACE: ${{ github.event.inputs.namespace }}
+          ACTIONS_ALLOW_UNSECURE_COMMANDS: 'true'
+        if: ${{ github.event.inputs.timestampMethod == 'delay' }}
+        run: |
+          chmod +x timestamp.sh
+          newtimestamp=`./timestamp.sh ${{ github.event.inputs.delay }}`
```

Comment:
Hmm, we can use bash script or put these commands here (and not use bash script execution), but I think it's up to us. What do you think @rryter ?

Reply:
i guess for modification purposes in the future, it's handy to have the script files

Reply:
ok, I agree with you @rryter , let's do this way :+1: